sdk/go/pulumi/context.go

```diff
@@ -50,6 +50,7 @@ type Context struct {
 	rpcsDone    *sync.Cond  // an event signaling completion of RPCs.
 	rpcsLock    *sync.Mutex // a lock protecting the RPC count and event.
 	rpcError    error       // the first error (if any) encountered during an RPC.
+	Log         Log
```

Comment:
Since this one is a public API - possibly add a comment on it - and perhaps even add a newline between this and the private fields?